workflows/cloudify_system_workflows/snapshot.py

```diff
@@ -477,24 +477,13 @@ def logstash_to_cloudify_events():
                 elem['_index'] = _EVENTS_INDEX_NAME
             yield elem
 
-    # cloudify_events -> logstash-*
-    def date_to_index_name(date):
-        d = datetime.strptime(date, '%Y-%m-%dT%H:%M:%S.%fZ')
-        return 'logstash-' + d.strftime('%Y.%m.%d')
-
-    def get_event_date(e):
-        return e['_source']['@timestamp']
-
     def cloudify_events_to_logstash():
-        event_indices = []
+        d = datetime.now()
+        index = 'logstash-{0}'.format(d.strftime('%Y.%m.%d'))
+        if not es.indices.exists(index=index):
```

Comment:
This is probably not needed (?). The index should get created automatically I think..